modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/TaskHeaderActions.tsx

```diff
@@ -59,12 +85,16 @@ const TaskHeaderActions: React.FC<TaskHeaderActionsProps> = ({
 				<ClayButton
 					displayType="secondary"
 					onClick={() =>
-						testrayTaskImpl.abandon(TestrayTask).then(mutateTask)
+						testrayTaskImpl
+							.abandon(testrayTask)
+							.then(mutateTask as any)
```

Comment:
This `as any` should not be here

```suggestion
							.then(mutateTask)
```